mlflow/store/model_registry/abstract_store.py

```diff
@@ -1,8 +1,18 @@
+import logging
 from abc import ABCMeta, abstractmethod
+from datetime import datetime, timedelta
+from time import sleep
 
+from mlflow.entities.model_registry.model_version_status import ModelVersionStatus
+from mlflow.exceptions import MlflowException
 from mlflow.utils.annotations import developer_stable
 
 
+_logger = logging.getLogger(__name__)
+
+AWAIT_MODEL_VERSION_CREATE_SLEEP_DURATION_SECONDS = 3
```

Comment:
```suggestion
AWAIT_MODEL_VERSION_CREATE_SLEEP_INTERVAL_SECONDS = 3
```

Reply:
Not sure why the original implementation uses 3 seconds, but is 3 seconds a reasonable sleep interval?

Reply:
I'm not sure if it is. Let me keep it for now.